pkg/plugins/policies/meshtcproute/api/v1alpha1/helpers.go

```diff
@@ -1,5 +1,11 @@
 package v1alpha1
 
+type PolicyDefault struct {
+	Rules []Rule
+}
+
 func (x *To) GetDefault() interface{} {
-	return x.Rules[0]
```

Comment:
One alternative could be returning the "default" rule, based on the `to.targetRef`?